packages/fluentui/docs/src/examples/components/Card/Variations/CardExampleSize.tsx

```diff
@@ -0,0 +1,111 @@
+          <Flex column>
+            <Text content="Small card" weight="bold" />
+            <Text content="Secondary line" size="small" />
+          </Flex>
+        </Flex>
+      </Card.Header>
+      <Card.Body>
+        <Flex column gap="gap.small">
+          <Image src="public/images/wireframe/square-image.png" fluid />
+          <Text content="Content text" />
+        </Flex>
+      </Card.Body>
+      <Card.Footer>
+        <Flex space="between">
+          <Button content="Action" />
+          <Flex>
+            <Button icon={<StarIcon />} iconOnly text title="Favourite" />
+            <Button icon={<MoreIcon />} iconOnly text title="More" />
+          </Flex>
+        </Flex>
+      </Card.Footer>
+    </Card>
+
+    <Divider />
+
+    <Card>
+      <Card.Header>
+        <Flex gap="gap.small">
+          <Avatar
+            image="public/images/avatar/small/matt.jpg"
+            label="Copy bandwidth"
+            name="Evie yundt"
+            status="unknown"
+          />
+          <Flex column>
+            <Text content="Medium (default) card" weight="bold" />
+            <Text content="Secondary line" size="small" />
+          </Flex>
+        </Flex>
+      </Card.Header>
+      <Card.Body>
+        <Flex column gap="gap.small">
+          <Image src="public/images/wireframe/square-image.png" fluid />
+          <Text content="Content text" />
+        </Flex>
+      </Card.Body>
+      <Card.Footer>
+        <Flex space="between">
+          <Button content="Action" />
+          <Flex>
+            <Button icon={<StarIcon />} iconOnly text title="Favourite" />
+            <Button icon={<DownloadIcon />} iconOnly text title="Download" />
+            <Button icon={<MoreIcon />} iconOnly text title="More" />
+          </Flex>
+        </Flex>
+      </Card.Footer>
+    </Card>
+
+    <Divider />
+
+    <Card size="large">
+      <Card.Header>
+        <Flex gap="gap.small">
+          <Avatar
+            image="public/images/avatar/small/matt.jpg"
+            label="Copy bandwidth"
+            name="Evie yundt"
+            status="unknown"
+          />
+          <Flex column>
+            <Text content="Large card" weight="bold" />
+            <Text content="Secondary line" size="small" />
+          </Flex>
+        </Flex>
+      </Card.Header>
+      <Card.Body>
+        <Flex column gap="gap.small">
+          <Image src="public/images/wireframe/square-image.png" fluid />
+          <Text content="Content text" />
+        </Flex>
+      </Card.Body>
+      <Card.Footer>
+        <Flex space="between">
+          <Button content="Action" />
+          <Flex>
+            <Button icon={<StarIcon />} iconOnly text title="Favourite" />
+            <Button icon={<DownloadIcon />} iconOnly text title="Download" />
+            <Button icon={<MoreIcon />} iconOnly text title="More" />
+          </Flex>
+        </Flex>
+      </Card.Footer>
+    </Card>
+  </Flex>
+);
+
+export default CardExampleSize;
```

Comment:
I suggest to remove `Body` or `Footer` from this example, too, to make it more simple